### ISSUE TITLE
Dashboard: to much PATCH view request, even when refresh rate is much higher. Causes high CPU load.

### ISSUE THREAD

**Issue by christian-beckmann** [influxdata/influxdb#23975]
__Steps to reproduce:__
List the minimal actions needed to reproduce the behavior.

1. Create a dashboard
2. Set Auto Refresh to 1h

__Expected behavior:__
The request for new data should only after 1h.

__Actual behavior:__
But in the browser network XHR view, it requests many PATCH to local:8086/api/v2/dashboards/123/cells/123/view every second.
Causes high CPU load.

__Environment info:__

Server: [5b6fdbf](https://github.com/influxdata/influxdb/tree/5b6fdbf05d)
InfluxDB 2.5.1
Frontend: [822e216](https://github.com/influxdata/ui/tree/822e216)

__Config:__
Copy any non-default config values here or attach the full config as a gist or file.

__Logs:__
Include snippet of errors in log.

__Performance:__
Generate profiles with the following commands for bugs related to performance, locking, out of memory (OOM), etc.

```sh
# Commands should be run when the bug is actively happening.
# Note: This command will run for ~30 seconds.
curl -o profiles.tar.gz "http://localhost:8086/debug/pprof/all?cpu=30s"
iostat -xd 1 30 > iostat.txt
# Attach the `profiles.tar.gz` and `iostat.txt` output files.
```

**Comment by niabassey** [comment]
I am having this issue even with Auto Refresh Off.   It doesn't happen every time but more often than not the interface gets laggy and console shows `Fetch finished loading: PATCH "<URL>".` 60 to 600 times per minute.

On occasions where the interface isn't lagging the fetch PATCH entries are also missing from console & network activity.

I have tried loading in an incognito window, logging in and out, and creating a new single value chart in a new dashboard but it still happens on most loads.

I did not have this issue on 2.3, it started after I upgraded the docker container to Influxdb 2.5.1.

**Comment by niabassey** [comment]
Additionally, when the dashboard is laggy it appears Configuring a Cell and then returning to the dashboard, either by saving changes or cancelling changes, resolves the issue, at least until the page is reloaded.

**Comment by jeffreyssmith2nd** [comment]
We have identified the Dashboard stability issue and rolled the UI back to a more stable version in the recent 2.6.0 release. Due to the number of issues related to this, I'm going to close all of these issues as Completed. Please open a new issue if you are still seeing the Dashboard stability problems on 2.6.0.
